src/Features/Core/Portable/Workspace/CompileTimeSolutionProvider.cs

```diff
@@ -205,57 +205,37 @@ private static bool IsRazorDesignTimeDocument(DocumentState documentState)
             var designTimeProjectDirectoryName = PathUtilities.GetDirectoryName(designTimeDocument.Project.FilePath)!;
 
             var generatedDocumentPath = BuildGeneratedDocumentPath(designTimeProjectDirectoryName, designTimeDocument.FilePath!, generatedDocumentPathPrefix);
-            var generatedDocumentPathNet6Preview7 = BuildGeneratedDocumentPathNet6Preview7(designTimeProjectDirectoryName, designTimeDocument.FilePath!, generatedDocumentPathPrefix);
```

Comment:
Will this break people using latest VS, but a global.json pointing to .NET 6 preview 7 or earlier?

Reply:
Yes, since .NET 6 SDK released no one should be using previews.